### DISCUSSION TITLE
Repositories bug

### DISCUSSION THREAD

**Discussion by culhamu** [community/community#43192] [Actions] [answered]
### Select Topic Area

Question

### Body

Although I sent nearly 20 classes to my github account yesterday and today, the boxes on my profile page say no contribution when I hover over today. The situation persists. How can I fix this?

**Comment by SilasRodrigues19** [comment, marked as answer]
Hey @culhamu 

When you click on the filter to show 2023 contributions, sometimes it doesn't show up and you have to refresh the page.

Access this link below, if it does not appear just refresh the page

https://github.com/culhamu?tab=overview&from=2023-01-01&to=2023-01-03

**Reply by culhamu** [reply]
.Thank you

**Comment by jfullstackdev** [comment]
what do you mean by sent nearly 20 classes to your GitHub account ? 

where did you contribute ? check this out 

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/viewing-contributions-on-your-profile#what-counts-as-a-contribution

and this 

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile

**Reply by culhamu** [reply]
I send a java question solution to my own repositories every day, but no contribution is visible in your contribution, what is the reason?

**Reply by culhamu** [reply]
I'm sending a class from Intellij daily but it still doesn't show, so I think of closing my account and reopening it. My friends don't have such a problem, they do almost what I do, but they show contribution and some of their boxes are dark green and some are light green. It only happens when I create new repositories.

**Reply by culhamu** [reply]
https://github.com/culhamu my page

**Comment by bilgep** [comment]
Created Repo in GitHub Repository Web UI > Pushed first commit with README.md file in it via Command Line > GitHub Repository Web UI still suggests adding a README file. Is this the expected behavior?
